javascript/phone.js

```diff
@@ -0,0 +1,69 @@
+(function () {
+  class Phone {
+    constructor(rootE) {
+      this.rootE = rootE;
+      this.conInput = this.rootE.querySelector(".phone__container__input");
+      this.checkBtn = this.rootE.querySelector(
+        ".phone__container__footer__btn--check"
+      );
+      this.clearBtn = this.rootE.querySelector(
+        ".phone__container__footer__btn--clear"
+      );
+      this.results = this.rootE.querySelector(".phone__container__results");
+
+      this.initEvents();
+    }
+
+    initEvents() {
+      this.checkBtn.addEventListener("click", () => {
+        this.checkValidNumber(this.conInput.value);
+        this.conInput.value = "";
+      });
+
+      this.conInput.addEventListener("keydown", (e) => {
+        if (e.key === "Enter") {
+          this.checkValidNumber(this.conInput.value);
+          this.conInput.value = "";
+        }
+      });
+
+      this.clearBtn.addEventListener("click", () => {
+        this.results.textContent = "";
+      });
+    }
+
+    checkValidNumber(input) {
+      if (input === "") {
+        alert("Please provide a phone number");
```

Comment:
Let's replace this alert with an HTML element that displays the message directly on the page.